webSocket/webSocketDriver.go

```diff
@@ -51,62 +51,59 @@ func NewWebsocketDriver(args ArgsWebSocketDriver) (*webSocketDriver, error) {
 
 // SaveBlock will send the provided block saving arguments within the websocket
 func (o *webSocketDriver) SaveBlock(outportBlock *outport.OutportBlock) error {
-	return o.handleAction(outportBlock, outportSenderData.OperationSaveBlock)
+	return o.handleAction(outportBlock, outportSenderData.PayloadSaveBlock)
```

Comment:
this is too specific to be part of the websocket package. This needs to be moved along with the constants definition

Reply:
refactored.